src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/interfaces/resources/ResourceProvider.java

```diff
@@ -0,0 +1,8 @@
+package com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.resources;
+
+public interface ResourceProvider {
```

Comment:
Point taken.  Although the concrete implementations may or may not provide "Internationalized Strings".  It could also provide hard-coded ones.  Or it could look them up in a database.  Or from a web service.  My design is so flexible it could handle any of that.  So probably "StringLiteralResourceProvider" would work better.

Perhaps I should have added interfaces "FizzStringLiteralProvider" and "BuzzStringLiteralProvider" to make it clearer what kinds of string literals you can get.